webapp/components/NewsList/NewsList.spec.ts

```diff
@@ -38,56 +40,45 @@ const newsItemsMock = [
 
 describe('NewsList', () => {
   const localVue = createLocalVue()
+  localVue.use(Vuex)
 
-  it('exists', () => {
-    const wrapper = mount(NewsList, {
-      localVue,
+  it('should exist', () => {
```

Comment:
Don't use `should`. Be affirmative and say `it exists`, period. Or maybe a more specific test case description.